import/trello/importTrello.ts

```diff
@@ -135,6 +136,29 @@ function convert(input: Trello): Block[] {
 
             outCard.fields.contentOrder = [text.id]
         }
+
+        // Add Checklists
+        if (card.idChecklists && card.idChecklists.length) {
+            card.idChecklists.forEach(checklistID => {
+                const lookup = input.checklists.find(e => e.id === checklistID)
+                if (lookup !== undefined) {
```

Comment:
In general, we can check if a variable is "tuthy" i.e. `if (lookup) {`, which will handle undefined, null, and other cases that can lead to a null reference exception.